pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -3,15 +3,22 @@
 , buildPythonPackage
 , fetchPypi
 , pythonOlder
-, async-timeout
+# install_requires
```

Comment:
I am not a fan of those comments. They don't really add anything and if requirements change they can get out of date quite easily. If you want to keep them, we can keep them though.

Reply:
They make it alot easier to compare dependencies, because you don't have to walk through the whole list of them.